os/booting-on-digitalocean.md

```diff
@@ -16,7 +16,7 @@ The following command will create a single droplet. For more details, check out
     <div class="tab-pane" id="alpha">
       <div class="channel-info">
         <p>The Alpha channel closely tracks master and is released frequently. The newest versions of system libraries and utilities will be available for testing. The current version is Container Linux {{site.alpha-channel}}.</p>
-        <a href="https://cloud.digitalocean.com/droplets/new?image=coreos-alpha" class="btn btn-default">Launch Container Linux Droplet</a><br/><br/>
+        <a href="https://cloud.digitalocean.com/droplets/new?distroImage=coreos-alpha" class="btn btn-default">Launch Container Linux Droplet</a><br/><br/>
```

Comment:
The old URL works, but the new one selects Ubuntu instead.

Reply:
Oops, I'll fix